### PR TITLE
Fix title hover on panel for video

### PR DESCRIPTION
**Pull request by yanfali**

### CODE REVIEW

File: src/i18n/en/index.js [hunk 1]
```diff
@@ -109,6 +109,7 @@ export default {
         },
         toggleTutorial: {
           label: 'Video Tutorial',
+          title: 'Mechmerlin howto video on Configurator',
```
Comment (noroadsleft) [inline src/i18n/en/index.js]:
```suggestion
          title: 'MechMerlin how-to video on Configurator',
```

File: src/i18n/en/index.js [hunk 2]
```diff
@@ -109,6 +109,7 @@ export default {
...
           help: 'Mechmerlin video guide'
```
Comment (noroadsleft) [inline src/i18n/en/index.js]:
```suggestion
           help: 'MechMerlin video guide'
```

**Comment by yanfali** [comment]
Thanks @noroadsleft

**Comment by noroadsleft** [comment]
Looks good to me.

**Pull request merged**